contracts/interfaces/IAxelarGMPExecutable.sol

```diff
@@ -24,7 +24,7 @@ interface IAxelarGMPExecutable {
      * @notice Returns the address of the AxelarGMPGateway contract.
      * @return The Axelar GMP Gateway contract associated with this executable contract.
      */
-    function gateway() external view returns (IAxelarGMPGateway);
+    function gateway() external view returns (IAxelarGMPGatewayWithToken);
```

Comment:
This is the GMPExecutable, it doesn't expose token send functionality. See `gatewayWithToken` instead

Reply:
updated in next revision.